File: calculator/app/src/main/java/com/example/calculator/MainActivity.java
package com.example.calculator;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.GridLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        RecyclerView recyclerView=findViewById(R.id.recyclerVie);
        recyclerView.setLayoutManager(new GridLayoutManager(this,3));
        String[] symbols={"1","2","3","4","5","6","7","8","9","0"};
recyclerView.setAdapter(new CalAdapter(symbols));
    }
}

File: QuizApp/app/src/main/java/com/example/quizapp/MainActivity.kt
package com.example.quizapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val etInput: EditText = findViewById(R.id.et_input)
        val btnStart: Button = findViewById(R.id.btn_start)
        btnStart.setOnClickListener {
            if (etInput.text.isEmpty()) {
                Toast.makeText(
                    this,
                    "Name is mandatory", Toast.LENGTH_LONG
                ).show()
            } else {
                val intent = Intent(this, QuizQuestionActivity::class.java)
                startActivity(intent)
                finish()
            }
        }
    }
}